content/en/platform/corda/4.13/enterprise/node/setup/corda-configuration-fields.md

```diff
@@ -346,8 +346,12 @@ Allows fine-grained controls of various features only available in the enterpris
 
 * `tuning`
 
-    * The Corda Node configuration file section that contains performance tuning parameters for Corda Enterprise Nodes.
+    * The Corda Node configuration file section that contains performance tuning parameters for Corda Enterprise nodes.
 
+    - `additionalFlowThreadPools`
+
+       * The default Corda configuration creates a single thread pool whose size is configured by the *[flowThreadPoolSize]({{< relref "#enterpriseconfiguration" >}})* parameter. You can define *multiple* thread pools and assign flows to them; for example, to prioritize particular flows and to segregate them from other flows. For more information, see [Setting thread pools]({{< relref "../../cordapps/thread-pools.md" >}}).
+       
```

Comment:
This is missing the threadpool and size sub params, also the fact that its an array.
What about also including the same example here as you did in the description page?